resources/gemrc.rb

```diff
@@ -46,6 +46,7 @@
     user new_resource.user if new_resource.property_is_set?("user")
     group new_resource.group if new_resource.property_is_set?("group")
     recursive true
+    not_if { new_resource.name == :local || new_resource.name == "local" }
```

Comment:
This can be simplified:

```ruby
not_if { new_resource.name.to_sym == :local }
```

Ditto on the `coerce_path` method:

```ruby
def coerce_path(value)
  case value.to_sym
  when :local
    ::File.join(Dir.home, ".gemrc")
  when :global
    Gem::ConfigFile::SYSTEM_WIDE_CONFIG_FILE
  else
    value
  end
end
```